app/posts/find-teacher-training/2021-08-18-visa-sponsorship-filter.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Visa sponsorship filter
+description: Why we added a new filter to Find
+date: 2021-08-18
+---
+
+Previous analysis showed us that a common reason for rejection is international candidates without the right to work or study in the UK applying for courses that are unable to sponsor visas.
```

Comment:
We can simplify this sentence to make it more accessible:

'Providers often reject candidates for not having the right to work or study in the UK. Candidates would apply for a course, not knowing if the provider can sponsor a visa.'

Reply:
I've reworded this a bit, and added some more detail:

> In the 2021 to 2022 cycle, providers often rejected candidates for not having the right to work or study in the UK. We think this is because it was hard for candidates to tell which courses were able to sponsor visas (they had to either ask the provider, or check a long list of providers with the ability to sponsor visas).

What do you think?